osu.Game/Overlays/Comments/ShowMoreRepliesButton.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Graphics;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Graphics;
+
+namespace osu.Game.Overlays.Comments
+{
+    public class ShowMoreRepliesButton : GetCommentRepliesButton
+    {
+        public ShowMoreRepliesButton(Comment comment)
+            : base(comment)
+        {
+            Margin = new MarginPadding { Vertical = 10, Left = 80 };
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            IdleColour = colours.Blue;
+            HoverColour = colours.BlueLighter;
+        }
+
+        protected override void OnChildrenChanged(ValueChangedEvent<List<Comment>> children)
+        {
+            Alpha = (!children.NewValue.Any() && Comment.RepliesCount > 0) || children.NewValue.Count == Comment.RepliesCount ? 0 : 1;
```

Comment:
I don't understand the first condition here. I assume the second is to cover the case where all the replies are loaded at once, but what's with the first?

Also in general I think this is too complicated - as far as I understand you *know* how many replies are expected in total, so if you can access the *total* number of children at the point of calling this, this should reduce to checking if the counts are equal.

Reply:
First condition is for the case, when we know, what there are replies exist, but they are no loaded and atm we don't have any children (in this case `LoadMoreCommentsButton` will be shown)